### PR TITLE
Added sprite-based glyphs to LineVis

### PR DESCRIPTION
**Pull request by loichuder**
image

### CODE REVIEW

File: src/h5web/visualizations/line/SpriteMaterial.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      attach="material"
+      map={sprite}
+      color={color}
+      size={6}
+      transparent
+    />
+  );
+}
+
+export default SpriteMaterial;
```
Comment (loichuder) [inline src/h5web/visualizations/line/SpriteMaterial.tsx:35]: Even though `Square` can be rendered by a simple `pointsMaterial`, I prefered to use a sprite to have a consistent behaviour for all `SpriteGlyph`.

File: src/h5web/visualizations/line/DataCurve.tsx [hunk 2]
```diff
@@ -27,16 +28,14 @@ function DataCurve(): JSX.Element {
     );
   }
 
-  if (glyph === Glyph.Square) {
-    return (
+  return (
+    <Suspense fallback={<></>}>
```
Comment (loichuder) [inline src/h5web/visualizations/line/DataCurve.tsx:32]: `Suspense` needed due to the `useLoader` in SpriteMaterial

File: src/h5web/visualizations/line/SpriteMaterial.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useLoader } from 'react-three-fiber';
+import { TextureLoader } from 'three';
+import { SpriteGlyph, Glyph } from './models';
+import circleURL from './sprites/circle.png';
+import crossURL from './sprites/cross.png';
+import squareURL from './sprites/square.png';
+
+const glyphURLs = {
```
Comment (axelboc) [inline src/h5web/visualizations/line/SpriteMaterial.tsx:9]: `GLYPH_URLS` ?

File: src/h5web/visualizations/line/SpriteMaterial.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useLoader } from 'react-three-fiber';
+import { TextureLoader } from 'three';
+import { SpriteGlyph, Glyph } from './models';
+import circleURL from './sprites/circle.png';
+import crossURL from './sprites/cross.png';
+import squareURL from './sprites/square.png';
```
Comment (axelboc) [inline src/h5web/visualizations/line/SpriteMaterial.tsx:7]: did you run them through https://tinypng.com/ ?

Comment (axelboc) [reply src/h5web/visualizations/line/SpriteMaterial.tsx:7]: isn't the cross a bit too thick, do you reckon?

**Pull request merged**